docs/references/nextjs/custom-signup-signin-pages.mdx

```diff
@@ -62,6 +62,22 @@ If Clerk's prebuilt components don't meet your specific needs or if you require
     ```
   </CodeBlockTabs>
 
+  ### Add custom /sign-in and /sign-up routes to middleware
+
+  `clerkMiddleware()` does not explicitly make `/sign-in` and `/sign-up` routes public, so we need to be sure they will be public for people to access the custom routes to sign in and sign up. In your `middleware.ts` file, update the following code to explicitly allow the `sign-in` and `sign-up` pages to be public. This will allow users to access the custom sign-in and sign-up pages, while still protecting the rest of your application's routes.
```

Comment:
nit: we don't use 'we' in the docs as per the Style guide. The reader is 'you' and Clerk is Clerk. 

This is getting better, but I think its still missing the when/why. Many customers will not need this, but this reads like all/most people will need it.

This needs something covering that this is for when all (or maybe most) routes are protected, so that users can still sign-up and sign-in.